htdocs/product/price.php

```diff
@@ -163,11 +163,15 @@
 			setEventMessages($object->error, $object->errors, 'errors');
 		}
 
-		if ($error)
-		{
+		if (!$error) {
```

Comment:
Question mark there : I reversed the condition as it made more sense as far as I could understand, but it would be best you checked if it's OK